api/dp-openapi.yaml

```diff
@@ -626,6 +751,33 @@ components:
           readOnly: true
       description: Statistics for the site
 
+    special_day:
+      type: object
+      required:
+      - id
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        comment:
+          type: string
+        url:
+          type: string
+          format: url
+        color:
+          type: string
+        symbol:
+          type: string
+        date_open:
```

Comment:
Why not keep month and date as separate fields? then the user could more easily format the date however they wanted to.

Reply:
Yeah, that's probably better. How does `date_open_month` and `date_open_day` sound as field names?

Reply:
Sounds alright to me.

Reply:
Works for me, too.

Reply:
Updated.